cJSON.c

```diff
@@ -1019,7 +1025,9 @@ CJSON_PUBLIC(cJSON *) cJSON_Parse(const char *value)
     return cJSON_ParseWithOpts(value, 0, 0);
 }
 
+#ifndef min
```

Comment:
Where would this `min` define come from? Do the Microsoft headers contain such a macro?

If there is a conflict it might be better to rename it to `cjson_min` or just inline it. Although inlining will make the code less readable.